toqito/channel_props/is_trace_preserving.py

```diff
@@ -0,0 +1,98 @@
+"""Is channel Trace-preserving."""
```

Comment:
The word `Trace` is not a proper noun, so I would suggest `Trace-preserving` should be changed to `trace-preserving`.